### ISSUE TITLE
Failure getting dependencies

### ISSUE THREAD

**Issue by ElonGaties** [moffatman/chan#199]
When running `flutter pub get` in the repo, during the 'Resolving Dependencies' process, I get the error `Could not find a file named "pubspec.yaml" in https://github.com/moffatman/flutter_photo_manager.git bbe70a4f6c001f1bc9cf4345b20cdb7d30450640.`

**Comment by moffatman** [comment]
Weird, because the file clearly exists

https://github.com/moffatman/flutter_photo_manager/blob/bbe70a4f6c001f1bc9cf4345b20cdb7d30450640/pubspec.yaml

Not sure how to help you with this. Maybe your git creds for GitHub are bad.

Anyway upon checking into this, I'll probably soon upgrade that dependency and remove my forked version. So maybe it will be fixed on its own then too, if this is the only dependency giving you trouble.

**Comment by ElonGaties** [comment]
Yeah, I really don't have a clue why it is throwing this either, I've tried on two machines (linux and windows) with the same resulting issue - I will fiddle with my git creds some more to see if I can find a fix.

Hopefully switching dependency does fix this however.
